plinth/views.py

```diff
@@ -30,7 +30,20 @@
 
 from . import forms, frontpage
 import plinth
+from django.utils.translation import ugettext_lazy as _
 
+from plinth import cfg
+
+
+def apps_init():
```

Comment:
I think this needs to go in ```__main__.py```. Perhaps it can be run before the modules are loaded (https://github.com/freedombox/Plinth/blob/master/plinth/__main__.py#L338). The same would apply to system module's init.